tezos/client/src/client.rs

```diff
@@ -116,6 +116,19 @@ pub fn call_protocol_json_rpc(request: ProtocolJsonRpcRequest) -> Result<JsonRpc
     }
 }
 
+/// Call compute path
```

Comment:
@adonagy 
can you please create jira for this to be rewritten in rust,
and put `// TODO: TE-XYZ...` here